install_config/build_defaults_overrides.adoc

```diff
@@ -35,19 +36,38 @@ values that have been configured for a specific build. However, if those values
 are not present on the build definition, they are set to the default value.
 
 * The `BuildOverrides` admission control plug-in allows administrators to
-override a setting in a build, regardless of the value stored in the build. It
-currently supports overriding the `forcePull` flag on a build strategy to
-enforce always refreshing the local image during a build by pulling the image
-from the registry. This ensures that a user can only build with an image that
+overrides a setting in a build, regardless of the value stored in the build. The plug-in
+currently supports xref:manually-setting-global-build-overrides[overriding the `forcePull` 
+flag on a build strategy] to force refreshing the local image from the registry during a build. 
+Refreshing ensures that users can build only with images that
 they are allowed to pull. The plug-in can also be configured to apply a set of
 image labels to every built image.
++
+For information on configuring the `BuildOverrides` admission control plug-in and
+the values you can override, see
+xref:manually-setting-global-build-overrides[Manually Setting Global Build Overrides].
+
+The default node selectors and the `BuildDefaults` or `BuildOverride` admission plugins work together as follows:
+
+* The default  project node selector, defined in the `projectConfig.defaultNodeSelector` field in the master configuration file,  
+is applied to the pods created in all projects without a specified `nodeSelector` value. This includes builds with `nodeSelector="null"` 
```

Comment:
"This includes builds" - I thought these settings applied to pods. Should it be "These settings are applied to builds?"